spec/unit/intercom/company_spec.rb

```diff
@@ -35,4 +35,10 @@
     _(proxy.url).must_equal '/companies/1/contacts'
     _(proxy.resource_class).must_equal Intercom::Contact
   end
+
+  it "deletes a company" do
+    company = Intercom::Company.new("id" => "1")
+    client.expects(:delete).with("/companies/1", {}).returns(test_company)
```

Comment:
Since we aren't checking for an expected return value, suggest we can drop the `.returns`
```suggestion
    client.expects(:delete).with("/companies/1", {})
```